apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.ts

```diff
@@ -493,95 +548,100 @@ function moveRowDown(event) {
   updateRubricItemOrderField();
 }
 
-function moveRowUp(event) {
+function moveRowUp(event: Event) {
   if (!(event.target instanceof HTMLElement)) return;
   const row = event.target.closest('tr');
   if (!row || !row.parentNode || !row.nextElementSibling || !row.previousElementSibling) return;
   row.parentNode.insertBefore(row.previousElementSibling, row.nextElementSibling);
   updateRubricItemOrderField();
 }
 
-function deleteRow(event) {
+function deleteRow(event: Event) {
   if (!(event.target instanceof HTMLElement)) return;
   const table = event.target.closest('table');
   event.target.closest('tr')?.remove();
   if (!table?.querySelectorAll('.js-rubric-item-row-order')?.length) {
-    table.querySelector('.js-no-rubric-item-note')?.classList.remove('d-none');
+    table?.querySelector('.js-no-rubric-item-note')?.classList.remove('d-none');
   }
   updateRubricItemOrderField();
   checkRubricItemTotals();
 }
 
-function rowDragStart(event) {
+function rowDragStart(event: JQuery.TriggeredEvent) {
   if (!(event.target instanceof HTMLElement)) return;
-  window.rubricItemRowDragging = event.target.closest('tr');
-  if (event.originalEvent?.dataTransfer) {
+  const closestRow = event.target.closest('tr');
+  if (closestRow) {
+    window.rubricItemRowDragging = closestRow;
+  }
+
+  if (!(event.originalEvent instanceof DragEvent)) return;
+  if (event.originalEvent.dataTransfer) {
     event.originalEvent.dataTransfer.effectAllowed = 'move';
   }
 }
 
-function rowDragOver(event) {
+function rowDragOver(event: Event) {
   if (!(event.target instanceof HTMLElement)) return;
   const row = event.target.closest('tr');
   // Rows in different tables don't count
-  if (!row || row.parent !== window.rubricItemRowDragging.parent) return;
+  if (!row || row.parentNode !== window.rubricItemRowDragging.parentNode) return;
```

Comment:
I am pretty sure that neither vanilla JS or jQuery has a `parent` property. jQuery has a `.parent` method -- maybe this is how it was working before?